specialScenarios/scenarioRiver.py

```diff
@@ -2,6 +2,7 @@ class scenarioRiver:
     def __init__(self, manager):
```

Comment:
This scenario should also include the logic that you put in the ferry scenario. They are a single scenario not two separate ones.